src/modify_resources.py

```diff
@@ -66,14 +70,15 @@ def reset_egress_network_bandwidth(ssh_client, container_id):
     _,_,err_val_rate = ssh_client.exec_command(reset_network_cmd)
     if len(err_val_rate.readlines()) != 0:
         print 'ERROR: Resetting of container id {} network failed'.format(container_id)
+        print 'ERROR MESSAGE: {}'.format(err_val_rate)
         return -1
     else:
         print 'SUCCESS: Stress of container id {} removed'.format(container_id)
         return 1
 
 # Fix me!
 # Removes all network manipulations for ALL machines in the Quilt Environment
```

Comment:
Fix this comment. Network Manipulation is not removed for all machines, just for the container_id specified.